### ISSUE TITLE
Allow arbitrary values in config file templates

### ISSUE THREAD

**Issue by patcon** [escapestudios-cookbooks/newrelic#218]
In particular, `nrsysmond.cfg`, in which I would like to disabled docker checks so that the logs don't have noisy errors:

https://docs.newrelic.com/docs/servers/new-relic-servers-linux/installation-configuration/configuring-servers-linux#cfgvar-disable_docker

Seems this would be useful for all the templates, so that this cookbook doesn't need to play catch-up when config params change

**Comment by ge1st** [comment]
I agree, I just found this issue because I'm looking to configure the `class_transformer` for the Java agent. As is, I have to copy the yaml file over and wrap this cookbook to use my cookbook's file instead of this one. Yay, now I have to maintain that template too.

Considering the config is just yaml this would be pretty simple.
